src/app/modules/feedback/feedback.error.ts

```diff
@@ -0,0 +1,17 @@
+import { ApplicationError } from '../core/core.errors'
+
+export class InvalidSubmissionIdError extends ApplicationError {
+  constructor(
+    message = 'Feedback submissionId can not be found in the form submissions. Please submit a form before you submit a feedback.',
```

Comment:
the user doesn't know what a `submissionId` is, so people who see this would likely be confused. perhaps we can just use a generic error message, like "Sorry, something went wrong. Please try again."?